Edit Tool Method Test Page Bug Fix

Error: src/app/editing-tool-test-page/editing-tool-test-page.page.html:15:66 - error TS2341: Property 'saveChangesToCloud' is private and only accessible within class 'EditingToolTestPagePage'.

15     <ion-button slot="end" color="light" fill="outline" (click)="saveChangesToCloud()" *ngIf="needSaving">
                                                                    ~~~~~~~~~~~~~~~~~~

  src/app/editing-tool-test-page/editing-tool-test-page.page.ts:14:16
    14   templateUrl: './editing-tool-test-page.page.html',
                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    Error occurs in the template of component EditingToolTestPagePage.


Error: src/app/editing-tool-test-page/editing-tool-test-page.page.html:77:32 - error TS2341: Property 'saveChangesToCloud' is private and only accessible within class 'EditingToolTestPagePage'.

77       <ion-fab-button (click)="saveChangesToCloud()">
                                  ~~~~~~~~~~~~~~~~~~

  src/app/editing-tool-test-page/editing-tool-test-page.page.ts:14:16
    14   templateUrl: './editing-tool-test-page.page.html',
                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    Error occurs in the template of component EditingToolTestPagePage.


Error: src/app/editing-tool-test-page/editing-tool-test-page.page.html:97:76 - error TS2341: Property 'onTitleEditorChange' is private and only accessible within class 'EditingToolTestPagePage'.

97         value="{{contents.segment[currentSeg].segmentTitle}}" (ionChange)="onTitleEditorChange()"
                                                                              ~~~~~~~~~~~~~~~~~~~

  src/app/editing-tool-test-page/editing-tool-test-page.page.ts:14:16
    14   templateUrl: './editing-tool-test-page.page.html',
                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    Error occurs in the template of component EditingToolTestPagePage.


Error: src/app/editing-tool-test-page/editing-tool-test-page.page.html:98:21 - error TS2341: Property 'titleFocus' is private and only accessible within class 'EditingToolTestPagePage'.

98         (ionFocus)="titleFocus()"></ion-input>
                       ~~~~~~~~~~

  src/app/editing-tool-test-page/editing-tool-test-page.page.ts:14:16
    14   templateUrl: './editing-tool-test-page.page.html',
                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    Error occurs in the template of component EditingToolTestPagePage.

--- a/Cyclops/src/app/editing-tool-test-page/editing-tool-test-page.page.ts
+++ b/Cyclops/src/app/editing-tool-test-page/editing-tool-test-page.page.ts
@@ -78,7 +78,7 @@ export class EditingToolTestPagePage implements OnInit {
     await alert.present();
   }
 
-  private loadEditorDataById() {
+  loadEditorDataById() {
     this.firebaseService.getDataByIdService(this.articleId).subscribe(
       e => {
         this.contents = {

--- a/Cyclops/src/app/editing-tool-test-page/editing-tool-test-page.page.ts
+++ b/Cyclops/src/app/editing-tool-test-page/editing-tool-test-page.page.ts
@@ -265,7 +265,7 @@ export class EditingToolTestPagePage implements OnInit {
 
   }
 
-  private async saveChangesToCloud() {
+  async saveChangesToCloud() {
     const alert = await this.alertController.create({
       cssClass: 'my-custom-class',
       message: 'Do you want to save all changes to Cloud?',

--- a/Cyclops/src/app/editing-tool-test-page/editing-tool-test-page.page.ts
+++ b/Cyclops/src/app/editing-tool-test-page/editing-tool-test-page.page.ts
@@ -259,7 +259,7 @@ export class EditingToolTestPagePage implements OnInit {
     }
   }
 
-  private reloadPage() {
+  reloadPage() {
     this.contents = null;
     this.loadEditorDataById();
 

--- a/Cyclops/src/app/editing-tool-test-page/editing-tool-test-page.page.ts
+++ b/Cyclops/src/app/editing-tool-test-page/editing-tool-test-page.page.ts
@@ -194,11 +194,11 @@ export class EditingToolTestPagePage implements OnInit {
     // this.needSaving = true;
     // console.log("need Saving on Title Editor Change");
   }
-  private titleFocus() {//change saving state to open when title input focused
+  titleFocus() {//change saving state to open when title input focused
     this.needSaving = true;
     console.log("title focus");
   }
-  private textAreaFocus() {
+  textAreaFocus() {
     this.needSaving = true;
     console.log("text area focus");
   }

--- a/Cyclops/src/app/editing-tool-test-page/editing-tool-test-page.page.ts
+++ b/Cyclops/src/app/editing-tool-test-page/editing-tool-test-page.page.ts
@@ -188,7 +188,7 @@ export class EditingToolTestPagePage implements OnInit {
     // console.log("Changes saved locally!");
   }
 
-  private onTitleEditorChange() {
+  onTitleEditorChange() {
     console.log("current title is: " + this.TitleInput);
     this.contents.segment[this.currentSeg].segmentTitle = this.TitleInput;
     // this.needSaving = true;

--- a/Cyclops/src/app/editing-tool-test-page/editing-tool-test-page.page.ts
+++ b/Cyclops/src/app/editing-tool-test-page/editing-tool-test-page.page.ts
@@ -61,7 +61,7 @@ export class EditingToolTestPagePage implements OnInit {
     this.presentAlert();
   }
 
-  private updateDataById(docId, data) {
+  updateDataById(docId, data) {
     this.firebaseService.updateDataByIdService(docId, data).then((res: any) => {
       console.log(res);
     }).catch((error) => {

--- a/Cyclops/src/app/editing-tool-test-page/editing-tool-test-page.page.ts
+++ b/Cyclops/src/app/editing-tool-test-page/editing-tool-test-page.page.ts
@@ -203,7 +203,7 @@ export class EditingToolTestPagePage implements OnInit {
     console.log("text area focus");
   }
 
-  private updateArticle() {
+  updateArticle() {
     this.editorComponent.editorInstance.setData(this.contents.segment[this.currentSeg].segmentBody)
   }
 

